bower.json

```diff
@@ -17,6 +17,6 @@
   ],
   "dependencies": {
     "bootstrap": ">=2.3.2",
-    "jquery": "~1.9.0"
+    "jquery": ">=1.9.0"
```

Comment:
This is the same as in `package.json`. If I change it here, should I do the same in `package.json`?